src-ocaml/vdom.ml

```diff
@@ -33,79 +27,87 @@ type 'msg property =
   | Data of string * string
   (* Event (name, userkey, callback) *)
   | Event of string * 'msg eventHandler * 'msg eventCache option ref
-  (* | Event of string * (Web.Event.t -> 'msg) *)
   | Style of (string * string) list
 
 type 'msg properties = 'msg property list
 
-(* type 'msg taggerCallbacks =
-  { renderToHtmlString : unit -> string
-  ; patchVNodesIntoElement : 'msg applicationCallbacks ref -> Web.Node.t
-  } *)
-
 type 'msg t =
   | CommentNode of string
   | Text of string
   (* Node (namespace, tagName, key, unique, properties, children)  *)
   | Node of string * string * string * string * 'msg properties * 'msg t list
-  (* | ArrayNode of string * string * string * string * 'msg property array * 'msg t array *)
   (* LazyGen (key, fnGenerator) *)
   | LazyGen of string * (unit -> 'msg t) * 'msg t ref
-  (* Tagger (toString, toDom, toVNodes) *)
-(* | Tagger of (unit -> string) * ('msg applicationCallbacks ref -> Web.Node.t -> Web.Node.t -> int ->  'msg t list -> Web.Node.t) * (unit -> 'msg t) *)
   (* Tagger (tagger, vdom) *)
   | Tagger of ('msg applicationCallbacks ref -> 'msg applicationCallbacks ref) * 'msg t
-  (*  *)
-  (* | Tagger of (('a -> 'msg) -> 'a t -> 'msg t) *)
-  (* Custom (key, cbAdd, cbRemove, cbChange, properties, children) *)
```

Comment:
Technically this one is going to come back once I get around to adding support for it in, can you keep it in so it acts as a reminder for me, maybe add a `TODO` comment for it for easily greppability?  :-)

Reply:
For note, I'm speaking of the `Custom` node, not anything else.  :-)